sympy/physics/control/lti.py

```diff
@@ -20,8 +20,42 @@ def _roots(poly, var):
         r = [rootof(poly, var, k) for k in range(n)]
     return r
 
+class LTICommon:
+    """A common class for all the Linear Time-Invariant Dynamical Systems."""
+    # Users should not directly interact with this class.
+    def __new__(cls, system, **kwargs):
+        if cls is LTICommon:
+            raise NotImplementedError('The LTICommon class is not meant to be used directly.')
+        obj = super(LTICommon, cls).__new__(cls)
+        obj.__var = system.var
+        obj.__num_inputs = getattr(system, 'num_inputs', None)
+        obj.__num_outputs = getattr(system, 'num_outputs', None)
+        return obj
```

Comment:
Avoid using double underscore private variables in Python as a general rule.

Avoid having any attributes in a Basic subclass. Presumably these private attributes are not used anywhere if they aren't used in this class so maybe just get rid of them.

The base classes should be passing their args to this class but it calls `super.__new__` without any args.